llvm/lib/Target/AMDGPU/AMDGPU.td

```diff
@@ -1110,6 +1110,13 @@ def FeatureRequiresCOV6 : SubtargetFeature<"requires-cov6",
   "Target Requires Code Object V6"
 >;
 
+def FeatureXF32Insts : SubtargetFeature<"xf32-insts",
+   "HasXF32Insts",
+   "true",
+   "Has instructions that support xf32 format, such as "
+   "v_mfma_f32_16x16x8xf32 and v_mfma_f32_32x32x4xf32"
```

Comment:
Typo?
```suggestion
   "v_mfma_f32_16x16x8_xf32 and v_mfma_f32_32x32x4_xf32"
```

Reply:
Alias :-)

`v_mfma_f32_16x16x8_xf32`, `v_mfma_f32_16x16x8xf32`, and `v_mfma_f32_16x16x8_xf32`, all the same thing.